packages/fields/src/types.ts

```diff
@@ -39,6 +48,11 @@ export interface BasePost extends CommonPost {
 	permalink_template?: string;
 	date?: string;
 	modified?: string;
+	author?: number;
+}
+
+export interface BasePostWithEmbeddedAuthor extends BasePost {
```

Comment:
Should this be `BasePostWithEmbeddedAuthor` or `BasePostWithEmbeds`. Basically what I'm asking is what are we doing at the request level is it something like `embed=true` or `embed[]=author`

Reply:
It's [_embed=author](https://github.com/WordPress/gutenberg/blob/6d36c1b8ee1c3c68601a61959bc1c2602ee107d1/packages/edit-site/src/components/post-list/index.js#L283), that's why I thought it'd be best to make it specific to that. Happy to rename it to `BasePostWithEmbeds`.